cms/static/cms/js/modules/cms.base.js

```diff
@@ -118,8 +120,8 @@ var CMS = {
             },
 
             // handles the tooltip for the plugins
-            showTooltip: function (name, id) {
-                var tooltip = $('.cms-tooltip');
+            showTooltip: function (e, name, id) {
+                var tooltip = CMS.API.Helpers.pickTooltip();
```

Comment:
it would actually be amazing if we could move tooltip init / related methods out of helpers (leaving old api of showTooltip / hideTooltip)  into it's own cms.tooltip.js